app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -28,30 +30,42 @@ IUSE=""
 
 DEPEND=">=dev-lang/go-1.3
 	app-arch/cpio
-	sys-fs/squashfs-tools"
+	sys-fs/squashfs-tools
+	dev-perl/Capture-Tiny"
 RDEPEND="!app-emulation/rocket"
 
-src_unpack() {
-	local cache="${S}/stage1/rootfs/usr_from_coreos/cache"
+src_configure() {
+	local myeconfargs=(
+		--with-stage1=host
```

Comment:
The quickest way forward probably would be to use the pre-built `coreos` (pairing rkt with a previously tested stage1 is perhaps safest anyway), we just need to skip the downloads and such and have the build trust the ebuild gives it the right thing. Longer term creating stage1 from an arbitrary $ROOT or at run-time is ideal.